plans/04-code-quality.md

```
@@ -31,7 +31,7 @@
 | H15 | Search has no debounce + pagination bug | High | [4](#4-h15--search-no-debounce--pagination-bug) |
 | M12 | `authRateLimitMiddleware` dead code | Medium | [5](#5-m12--authratelimitmiddleware-dead-code) |
 | M13 | Password strength is length-only | Medium | [6](#6-m13--password-strength-length-only) |
-| M15 | 10 stale Prisma references in docs | Medium | [7](#7-m15--stale-prisma-references-in-docs) |
+| M15 | 14+ stale Prisma refs in docs + README | Medium | [7](#7-m15--stale-prisma-references-in-docs) |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)